### DISCUSSION TITLE
student developer pack website

### DISCUSSION THREAD

**Discussion by Susmita14** [community/community#77204] [GitHub Education]
student developer pack in that website ,i am fill the form and click the continue button .but this button is not working proper .because  i click the button and then again visit the same page.so why this problem again created

**Comment by queenofcorgis** [comment]
Welcome to the GitHub Community, @Susmita14 , we're happy you're here! You are more likely to get a useful response if you are posting your question in the applicable category. I've gone ahead and moved it for you. Good luck!

**Comment by aslisubhash** [comment]
> student developer pack in that website ,i am fill the form and click the continue button .but this button is not working proper .because i click the button and then again visit the same page.so why this problem again created

Visit from your college it will work

**Comment by hackelite01** [comment]
I've noticed some challenges regarding the approval of the GitHub Student Developer Pack, and I'd like to offer my assistance. If you're facing issues or have questions, please feel free to drop your queries and details in my email at hackelite.sup@gmail.com. I'll do my best to help you navigate through the process and get your GitHub Student Developer Pack approved smoothly.
